packages/react-core/src/components/NotificationDrawer/NotificationDrawer.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+import styles from '@patternfly/react-styles/css/components/NotificationDrawer/notification-drawer';
+import { css } from '@patternfly/react-styles';
+
+export interface NotificationDrawerProps extends React.HTMLProps<HTMLDivElement> {
+  /**  Content rendered inside the Notification Drawer */
```

Comment:
Can you update this to be sentence case.  So it should be "notification drawer".